lib/test.js

```diff
@@ -104,51 +112,52 @@ Test.prototype.plan = function (count) {
 };
 
 Test.prototype.run = function () {
-	this.promise = {};
+	var self = this;
+
+	this.promise = Promise.pending();
```

Comment:
@BarryThePenguin we need to return a promise by the end of function and resolve/reject it later, so we can't use these right away.